libpacaur/checks.sh

```diff
@@ -27,13 +27,15 @@ IgnoreChecks() {
 
     checkaurpkgsAver=($(GetJson "var" "$json" "Version"))
     checkaurpkgsQver=($(expac -Q '%v' "${checkaurpkgs[@]}"))
+    # set always the latest revision for devel packages
```

Comment:
Yes. This is done because the version commit we can retrieve through the RPC is most likely wrong (outdated).